compare_core_with_daps_outputs/compare_common_entries.py

```diff
@@ -0,0 +1,210 @@
+# %%
+# Check EC2 connection
+!ec2metadata
+
+# %%
+repo_path = '/home/ubuntu/sky_workdir/asf_exploration/'  # path to asf_exploration folder
+
+# %%
+import sys
+sys.path.append(repo_path)
+
+import polars as pl
+pl.enable_string_cache()
```

Comment:
Just double checking if you're sure to require global string cache

Reply:
It was recommended to set it when I encountered an error running the script.